report/sections/data.tex

```diff
@@ -0,0 +1,71 @@
+\section{Data}\label{sec:data}
```

Comment:
You're right. I changed it before I added in the two fruit images (left them out originally because it seemed weird to show two identical images). I think the PSNR comparisons, which are 2/3 of the things discussed in the section, make "data" more descriptive than "examples."